train.py

```diff
@@ -61,6 +61,14 @@ def parse_arguments():
                         default=1e-4,
                         help='Optimization learning rate')
 
+    parser.add_argument('-ci',
+                        '--checkpoint-interval',
+                        dest='checkpoint_interval',
+                        action='store',
+                        type=float,
+                        default=1e-4,
+                        help='Optimization learning rate')
```

Comment:
type, default, and help do not look right.

Reply:
You are correct. Changing to `type=int`, `default=10`